tests/test_models.py

```diff
@@ -0,0 +1,75 @@
+from osier import DispatchModel
+from osier import Technology
+from unyt import unyt_array
+import numpy as np
+import pytest
+import sys
+
+if "win32" in sys.platform:
+    solver = 'cplex'
+elif "linux" in sys.platform:
+    solver = "cbc"
+
+TOL = 1e-5
+
+@pytest.fixture
+def technology_set():
+    nuclear = Technology(technology_name='Nuclear',
+                         technology_type='production',
+                         capacity=1e3,
+                         capital_cost=6,
+                         om_cost_variable=20,
+                         om_cost_fixed=50,
+                         fuel_cost=5
+                         )
+    natural_gas = Technology(technology_name='NaturalGas',
+                             technology_type='production',
+                             capacity=1e3,
+                             capital_cost=1,
+                             om_cost_variable=12,
+                             om_cost_fixed=30,
+                             fuel_cost=20
+                             )
+
+    return [nuclear, natural_gas]
+
+
+@pytest.fixture
+def net_demand():
+    n_hours = 24
+    np.random.seed(123)
```

Comment:
Interesting, I didn't realize it was being deprecated. The `seed` enables reproducibility for randomly generated values. 
That way, when I call "random" it will generate the same "random" values for you and for me.
There's a "better" syntax, though I'm not really sure why it's better. If you want to `osier` to adopt it, I encourage you to make a PR :).